tests/fsharp/typecheck/sigs/neg02.vsbsl

```diff
@@ -7,4 +7,4 @@ neg02.fs(17,21,17,26): typecheck error FS3351: Feature 'static abstract interfac
 
 neg02.fs(17,21,17,26): typecheck error FS3350: Feature 'static abstract interface members' is not available in F# 6.0. Please use language version 7.0 or greater.
 
-neg02.fs(17,21,17,24): typecheck error FS0855: No abstract or interface member was found that corresponds to this override
```

Comment:
Is this change related to this PR ?. Seem like this is what is making fail the CI

Reply:
Seems like bad merge?

Reply:
Good catch, adjusting it back.